test/ecto_enum_test.exs

```diff
@@ -287,6 +287,21 @@ defmodule EctoEnumTest do
               ]}
   end
 
+  test "generates correct t() typespec for postgres types when enums are strings" do
+    assert Code.Typespec.fetch_types(EctoEnum.Typespec.TestModule.PGRoleEnum) ==
+             {:ok,
+              [
+                type:
+                  {:t,
+                   {:type, 0, :union,
+                    [
+                      {:atom, 0, :user},
```

Comment:
should the typespec be atoms when we define strings?

Reply:
Unfortunately we can't define strings in typespec